src/assets/pages/blogposts/post-48.html

```diff
@@ -0,0 +1,31 @@
+<img src="https://querido-diario-static.nyc3.cdn.digitaloceanspaces.com/post0048/post-48-grafico-2.png">
+<p><span style="font-weight: 400;">Por&eacute;m, outra novidade este ano foi a maior aten&ccedil;&atilde;o dada aos reposit&oacute;rios do Querido Di&aacute;rio que anteriormente n&atilde;o costumavam receber tantas contribui&ccedil;&otilde;es no &acirc;mbito da Hacktoberfest. &Eacute; o caso do </span><a href="https://github.com/okfn-brasil/querido-diario-frontend"><span style="font-weight: 400;">querido-diario-frontend</span></a><span style="font-weight: 400;">, que guarda o c&oacute;digo do site, e tamb&eacute;m do </span><a href="https://github.com/okfn-brasil/querido-diario-comunidade"><span style="font-weight: 400;">querido-diario-comunidade</span></a><span style="font-weight: 400;">, onde fica concentrada toda a documenta&ccedil;&atilde;o geral e discuss&otilde;es transversais aos v&aacute;rios reposit&oacute;rios do projeto e sua comunidade.&nbsp;</span></p>
+<p><span style="font-weight: 400;">Al&eacute;m dos </span><em><span style="font-weight: 400;">pull requests (PRs)</span></em><span style="font-weight: 400;">, os reposit&oacute;rios do QD tamb&eacute;m foram movimentados com a cria&ccedil;&atilde;o de </span><em><span style="font-weight: 400;">Issues</span></em><span style="font-weight: 400;">, que representam demandas espec&iacute;ficas que as pessoas contribuidoras notam como necess&aacute;rias para melhorar o projeto.</span></p>
+<p><strong>Segunda centena de munic&iacute;pios.</strong><span style="font-weight: 400;"> Por conta das contribui&ccedil;&otilde;es no desenvolvimento de raspadores, uma grande conquista da participa&ccedil;&atilde;o na Hacktoberfest deste ano foi a </span><strong>integra&ccedil;&atilde;o de mais de 100 cidades ao Querido Di&aacute;rio.</strong><span style="font-weight: 400;"> Como compara&ccedil;&atilde;o, nesse mesmo per&iacute;odo no ano passado, </span><a href="https://ok.org.br/noticia/festival-de-contribuicoes-veja-o-impacto-do-hacktoberfest-nos-projetos-da-okbr/"><span style="font-weight: 400;">o projeto tinha cerca de 50 cidades</span></a><span style="font-weight: 400;"> e contribui&ccedil;&otilde;es para cerca de 40 novas cidades foram feitas em outubro, das quais 9 foram integradas ao longo do mesmo m&ecirc;s. Neste ano, iniciamos a Hacktoberfest com 118 munic&iacute;pios cadastrados na plataforma e chegamos ao dia 31 com 231!</span></p>
+<p><strong><em>[Confira lista das novas cidades no sininho no canto superior direito do <a href="https://queridodiario.ok.org.br/blog">site do Querido Di&aacute;rio</a>].&nbsp;</em></strong></p>
```

Comment:
O link seria pra página inicial e não `/blog`
```suggestion
<p><strong><em>[Confira lista das novas cidades no sininho no canto superior direito do <a href="https://queridodiario.ok.org.br/" target="_blank">site do Querido Di&aacute;rio</a>].&nbsp;</em></strong></p>
```

Reply:
foi corrigido em atendido em https://github.com/okfn-brasil/querido-diario-frontend/pull/227/commits/decc4930efe70c0e2fe2351859d94ba930afa64c